drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -126,13 +126,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
         Hmis::File.all_available_tags.map do |tag|
           {
             code: tag.id,
-            label: "#{tag.name} (included: #{tag.included_info})",
+            label: tag.name,
             group_code: tag.group,
             group_label: tag.group,
+            secondary_label: "(included: #{tag.included_info})",
           }
         end.
           compact.
           sort_by { |obj| [obj[:group_label] + obj[:label]].join(' ') }
+      when 'CLIENT_ENROLLMENTS'
+        client = Hmis::Hud::Client.viewable_by(user).find_by(id: relation_id)
+        return [] unless client.present?
+
+        client.enrollments.map do |enrollment|
+          {
+            code: enrollment.id,
+            label: enrollment.project.project_name,
```

Comment:
this might want to include the `entry_date` or have a `.uniq` since a client may be enrolled at the same project multiple times.

Reply:
I hesitate to exclude any enrollments unless there is another mechanism to attach them that doesn't involve the pick list (like go to the intake for that enrollment and add a file.)  I'm not sure what the use case would be for attaching a file to an old enrollment, but there may be one.  It'd probably be safe to limit to open enrollments and those closed within the past year.

Reply:
Okay, sounds good. @ZachCMP lets sort the enrollments using the `:most_recent` sort option, so it's easier to find recent enrollments.